test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -58,7 +58,8 @@
 
     (doseq [res result]
       (is (= #{:certname :deactivated :expired :catalog_timestamp :facts_timestamp :report_timestamp
-               :catalog_environment :facts_environment :report_environment} (keyset res))
+               :catalog_environment :facts_environment :report_environment
+               :latest_report_status :latest_report_hash} (keyset res))
```

Comment:
Seems like we should have a basic test that uses latest_report_status and latest_report_hash as query criteria